c/contracts.h

```diff
@@ -15,41 +15,42 @@
-#define BN256_PAIRING_PERPOINT_GAS_ISTANBUL  34000  // Per-point price for an elliptic curve pairing check
+#define SHA256_BASE_GAS 60        // Base price for a SHA256 operation
+#define SHA256_PERWORD_GAS 12     // Per-word price for a SHA256 operation
+#define RIPEMD160_BASE_GAS 600    // Base price for a RIPEMD160 operation
+#define RIPEMD160_PERWORD_GAS 120 // Per-word price for a RIPEMD160 operation
+#define IDENTITY_BASE_GAS 15      // Base price for a data copy operation
+#define IDENTITY_PERWORD_GAS 3    // Per-work price for a data copy operation
+
+#define BN256_ADD_GAS_BYZANTIUM 500                // Byzantium gas needed for an elliptic curve addition
+#define BN256_ADD_GAS_ISTANBUL 150                 // Gas needed for an elliptic curve addition
+#define BN256_SCALAR_MUL_GAS_BYZANTIUM 40000       // Byzantium gas needed for an elliptic curve scalar multiplication
+#define BN256_SCALAR_MUL_GAS_ISTANBUL 6000         // Gas needed for an elliptic curve scalar multiplication
+#define BN256_PAIRING_BASE_GAS_BYZANTIUM 100000    // Byzantium base price for an elliptic curve pairing check
+#define BN256_PAIRING_BASE_GAS_ISTANBUL 45000      // Base price for an elliptic curve pairing check
+#define BN256_PAIRING_PERPOINT_GAS_BYZANTIUM 80000 // Byzantium per-point price for an elliptic curve pairing check
+#define BN256_PAIRING_PERPOINT_GAS_ISTANBUL 34000  // Per-point price for an elliptic curve pairing check
 
 #define BLAKE2F_INPUT_LENGTH 213
 #define BLAKE2F_FINAL_BLOCK_BYTES 0x1
 #define BLAKE2F_NON_FINAL_BLOCK_BYTES 0x0
 
 /* pre-compiled Ethereum contracts */
 
-typedef int (*precompiled_contract_gas_fn)(const uint8_t* input_src,
+typedef int (*precompiled_contract_gas_fn)(const uint8_t *input_src,
                                            const size_t input_size,
-                                           uint64_t* gas);
-typedef int (*precompiled_contract_fn)(gw_context_t* ctx,
-                                       const uint8_t* code_data,
+                                           uint64_t *gas);
+typedef int (*precompiled_contract_fn)(gw_context_t *ctx,
+                                       const uint8_t *code_data,
                                        const size_t code_size,
                                        bool is_static_call,
-                                       const uint8_t* input_src,
+                                       const uint8_t *input_src,
                                        const size_t input_size,
-                                       uint8_t** output, size_t* output_size);
+                                       uint8_t **output, size_t *output_size);
 
-int ecrecover_required_gas(const uint8_t* input, const size_t input_size,
-                           uint64_t* gas) {
+int ecrecover_required_gas(const uint8_t *input, const size_t input_size,
+                           uint64_t *gas)
+{
```

Comment:
Why do those kind of style changes? Are those changes by hand or tool? If by tool please enforce it in Makefile and CI config.

Reply:
This is from `make fmt`. Should I revert back?

Reply:
Better change it in a separated PR, otherwise it a distraction when review the changes.